packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -0,0 +1,272 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const error = require('../error');
+const isA = require('joi');
+const ScopeSet = require('fxa-shared').oauth.scopes;
+
+module.exports = (log, db, config, customs, oauthdb, subhub) => {
+  const totpUtils = require('./utils/totp')(log, config, db);
+
+  const SUBSCRIPTIONS_MANAGEMENT_SCOPE =
+    'https://identity.mozilla.com/apps/subscriptions';
```

Comment:
I imagine we'll want to limit the ability of this scope to be requested by arbitrary RPs. Perhaps we only allow the content server and/or the subscription server to request this scope and we can relax the rule if necessary later.

Reply:
Yeah, this scope should only ever really be granted to the payment pages. I'm not sure how to restrict that, though.

Reply:
FWIW, I think this is worth punting off to a follow up issue:

https://github.com/mozilla/fxa/issues/881